common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -1926,27 +1926,40 @@
-  "service.wynntils.itemFilter.filter.ranged.name": "Ranged",
-  "service.wynntils.itemFilter.filter.ranged.usage": "Specify a single number (\"5\"), or a range (\"5-10\" or \">=13\" or \"<25\")",
+  "service.wynntils.itemFilter.filter.rangedCappedValue.description": "Filters whether a CappedValue's current value is in the filter range",
+  "service.wynntils.itemFilter.filter.rangedCappedValue.name": "Ranged (CappedValue)",
+  "service.wynntils.itemFilter.filter.rangedCappedValue.usage": "Specify a single number (\"5\"), or a range (\"5-10\" or \">=13\" or \"<25\")",
+  "service.wynntils.itemFilter.filter.rangedInteger.description": "Filters whether an integer is in the filter range",
+  "service.wynntils.itemFilter.filter.rangedInteger.name": "Ranged (Integer)",
+  "service.wynntils.itemFilter.filter.rangedInteger.usage": "Specify a single number (\"5\"), or a range (\"5-10\" or \">=13\" or \"<25\")",
   "service.wynntils.itemFilter.filter.string.description": "Filters whether a string matches the filter string",
   "service.wynntils.itemFilter.filter.string.name": "String",
   "service.wynntils.itemFilter.filter.string.usage": "\"profession: wood\" would match both woodcutting and woodworking. Using \"\" enables strict matching.",
   "service.wynntils.itemFilter.invalidFilter": "%s: invalid filter for type: %s",
+  "service.wynntils.itemFilter.stat.countedItem.description": "Count of an item (e.g. health potion), this is not the same as the amount of items in a stack.",
+  "service.wynntils.itemFilter.stat.durability.description": "Durability of the item",
+  "service.wynntils.itemFilter.stat.emeraldValue.description": "Emerald value of the item",
+  "service.wynntils.itemFilter.stat.gearRestriction.description": "Gear restriction of the item",
+  "service.wynntils.itemFilter.stat.gearType.description": "Gear type of the item (e.g. helmet)",
+  "service.wynntils.itemFilter.stat.itemType.description": "Item type of the item (e.g. craftedGearItem)",
   "service.wynntils.itemFilter.stat.level.description": "Level of the item",
-  "service.wynntils.itemFilter.stat.level.name": "Level",
+  "service.wynntils.itemFilter.stat.majorId.description": "Major ID(s) of the item",
+  "service.wynntils.itemFilter.stat.powderSlots.description": "Powder slots of the item",
   "service.wynntils.itemFilter.stat.profession.description": "List of professions related to the item",
-  "service.wynntils.itemFilter.stat.profession.name": "Profession",
   "service.wynntils.itemFilter.stat.quality.description": "Quality of the item",
-  "service.wynntils.itemFilter.stat.quality.name": "Quality",
   "service.wynntils.itemFilter.stat.rarity.description": "Rarity of the item",
-  "service.wynntils.itemFilter.stat.rarity.name": "Rarity",
+  "service.wynntils.itemFilter.stat.target.description": "Target of the item (e.g. Detlas)",
+  "service.wynntils.itemFilter.stat.tier.description": "Tier of the item (e.g. VI)",
```

Comment:
I think this might be a bit misleading as it suggests using VI to search for it. Maybe change to something like "e.g. tier:6 for tier VI items"